aexpect/client.py

```diff
@@ -331,8 +331,10 @@ def get_output(self):
         Return the STDOUT and STDERR output of the process so far.
         """
         try:
-            with open(self.output_filename, 'r') as output_file:
-                return output_file.read()
+            with open(self.output_filename, 'rb') as output_file:
+                errors = ('replace' if sys.version_info[0] < 3 else
+                          'backslashreplace')
+                return output_file.read().decode(self.encoding, errors)
```

Comment:
I could work on this but we also just hit an issue with this unicode enforcement on binary data and it would be great if we could merge this PR sooner.

Reply:
Howdy folks :)

For the python 2/3 specific code - How important is it to maintain it? Shouldn't we go all the way python2?

Reply:
Hi @lmr, we maintain python 2 mostly because of the Avocado VT plugin which makes good use of aexpect and still has a support for it. Until we resolve the need to drop python 2 there, I am afraid there is not much we can do here without breaking their workflow.